code/drasil-example/gamephysics/lib/Drasil/GamePhysics/Body.hs

```diff
@@ -155,7 +155,11 @@ symbMap = cdb (map (^. output) iMods ++ map qw symbolsAll) (nw gamePhysics :
   ++ [nw algorithm] ++ map nw derived ++ map nw fundamentals
   ++ map nw CM.mathcon ++ map nw CM.mathcon')
   (map cw defSymbols ++ srsDomains ++ map cw iMods) units dataDefs
-  iMods generalDefns tMods concIns section [] (offShelfSolRefs ++ [pymunk])
+  iMods generalDefns tMods concIns section [] (offShelfSolRefs ++ [pymunk] ++ allRefs)
+
+  -- | Holds all references and links used in the document.
+allRefs :: [Reference]
+allRefs = [externalLinkRef]
```

Comment:
You should probably append `offShelfSolRefs` and `[Pymunk]` when you construct `allRefs`, instead of doing it in the `symbMap` function.

```Haskell
allRefs :: [Reference]
allRefs = [externalLinkRef, pymunk] ++ offShelfSolRefs
```

Reply:
Okay, I ran into a conflict last time I tried to merge, and these `offShelfSolRefs` seem to be exclusive to the GamePhysics module.